production_configs/20240821_v0.10.11_src14_dec2276_tuned/lstmcpipe_config_2024-08-21_PathConfigAllSkyFullDL1ab.yaml

```diff
@@ -0,0 +1,196 @@
+# lstmcpipe generated config from PathConfigAllSkyFullDL1ab - 2024-08-21
+
+workflow_kind: lstchain
+
+# prod_id ex: local_no_n_islands. Default; v00 (if key left empty or None)
+prod_id: 20240821_v0.10.11_src14_dec2276_tuned
+
+source_environment:
+  source_file: /fefs/aswg/software/conda/etc/profile.d/conda.sh
+  conda_env: lstchain-v0.10.11
+
+slurm_config:
+# dpps is the default account for lstanalyzer - other users should use aswg
+  user_account: dpps
+lstmcpipe_version: 0.10.4
+prod_type: PathConfigAllSkyFullDL1ab
+stages_to_run:
+- dl1ab
+- merge_dl1
+- train_pipe
+- dl1_to_dl2
+- dl2_to_irfs
+stages:
+  dl1ab:
+  - input: 
+      /fefs/aswg/data/mc/DL1/AllSky/20240821_v0.10.11_src14_dec2276_tuned/TrainingDataset/dec_2276/GammaDiffuse/node_corsika_theta_66.446_az_75.983_
```

Comment:
Hi Ruben
Thanks for the PR.

Some issues should be addressed. This is a dl1ab production, but the starting point (DL1 input) does not exist (the config here would start from `/fefs/aswg/data/mc/DL1/AllSky/20240821_v0.10.11_src14_dec2276_tuned/` but that does not exist yet).
The mistake comes from the source prod ID in the command. You could start from prod `20240131_allsky_v0.10.5_all_dec_base` as base one and the command would be:

```
lstmcpipe_generate_config PathConfigAllSkyFullDL1ab --dec_list dec_2276 --prod_id 20240821_v0.10.11_src14_dec2276_tuned --kwargs source_prod_id=20240131_allsky_v0.10.5_all_dec_base
```

Reply:
ups, apologies! Thanks for the correct kwargs, I updated the files and checked that the input folders exist